js/anattadesign/awesomecheckout/opcheckout.js

```diff
@@ -1763,3 +1763,21 @@ jQuery.validator.addClassRules({
 		creditcard2: ''
 	}
 });
+
+// Additions to support MasterCard series 2
+var cc2 = jQuery.validator.methods['creditcard2'];
+jQuery.validator.addMethod('creditcard2', function (value, element, param) {
+
+    var cardName = param;
+
+    if (typeof(cardName) !== 'string' || typeof(cardName) == 'string' && cardName == '') {
```

Comment:
Yeah, it's needed. We fall back to using `ccType` to determine the type if the `cardName` param is not a string OR if it is an empty string. The order of operations is this:
* First section (typeof... is not "string")
* OR second section: (typeof is string but is empty)
   * This is because AND has a higher precedence than OR, so the last two expressions are grouped